git_machete/cli.py

```diff
@@ -156,6 +156,10 @@ def create_cli_parser() -> argparse.ArgumentParser:
     anno_parser.add_argument(
         '-H', '--sync-github-prs', action='store_true', default=argparse.SUPPRESS)
 
+    clean_parser = subparsers.add_parser('clean', usage=argparse.SUPPRESS, add_help=False, parents=[common_args_parser])
+    clean_parser.add_argument('-c', '--checkout-my-github-prs', action='store_true', default=argparse.SUPPRESS)
```

Comment:
After a consideration, I don't like using `-c` for this option... how about `-H` for consistency with `git machete anno -H`? consider the case where a support for other VCS is added (bitbucket/gitlab)... then having `-H/-B/-L` makes more sense then `-c/-B/-L`